pkgs/by-name/op/openvpn3/package.nix

```diff
@@ -0,0 +1,133 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, asio
+, glib
+, jsoncpp
+, libcap_ng
+, libnl
+, libuuid
+, lz4
+, openssl
+, pkg-config
+, protobuf
+, python3
+, systemd
+, tinyxml-2
+, wrapGAppsHook3
+, gobject-introspection
+, meson
+, ninja
+, gdbuspp
+, cmake
+, git
+, enableSystemdResolved ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openvpn3";
+  # also update openvpn3-core
+  version = "22_dev";
+
+  src = fetchFromGitHub {
+    owner = "OpenVPN";
+    repo = "openvpn3-linux";
+    rev = "refs/tags/v${version}";
+    # Don't forget to actualize version scripts in `postPatch`
+    hash = "sha256-UbphN5gHgO30ry0kX6W4qSs5Ksrvfhm4xiRNBdzEOhA=";
+    # `openvpn3-core` is a submodule.
+    # TODO: make it into a separate package
+    fetchSubmodules = true;
+  };
+
+  patches = [
+    ./patches/0001-customizable-asio-path.patch
+    ./patches/0002-customizable-installation-paths.patch
+  ];
+
+  postPatch = ''
+    echo '#define OPENVPN_VERSION "3.git:unknown:unknown"' > ./src/build-version.h
+    echo '#define PACKAGE_GUIVERSION "${builtins.replaceStrings ["_"] [":"] version}_unknown__s"' >> ./src/build-version.h
+    echo '#define PACKAGE_NAME "openvpn3-linux"' >> ./src/build-version.h
+
+    patchShebangs ** /*.py ** /*.sh \
```

Comment:
```suggestion
    patchShebangs **/*.py **/*.sh \
```
do you mean this?

Reply:
This was part of the original `openvpn3` (when it rested in `pkgs/tools`) thus I have kept it as-is, but I will do a re-check.

Reply:
UPD: I've actualized the call to `patchShebangs`, now only patches files which actually need it